infrastructure/smart-contracts/contracts/upgrades/referee/Referee9.sol

```diff
@@ -503,7 +495,17 @@ contract Referee9 is Initializable, AccessControlEnumerableUpgradeable {
         }
                 
         // we need to determine how much token will be emitted
-        (uint256 challengeEmission,) = calculateChallengeEmissionAndTier();
+        (uint256 emissionsPerHour, ) = calculateChallengeEmissionAndTier();
+
+        // Get the timestamp of the start of the current challenge
+        uint256 startTs;
+        if (challengeCounter == 0) {
+            startTs = block.timestamp - 3600; //1 hour
+        } else {
+            startTs = challenges[challengeCounter - 1].createdTimestamp;
+        }
+
+        uint256 challengeEmission = emissionsPerHour * (block.timestamp - startTs) / 3600; // Calculate the total emissions for the challenge based on the time passed
```

Comment:
Had to remove this local variable that stored time passed and do the calculation inline due to stack too deep on compilation.